test/e2e/managed_test.go

```diff
@@ -0,0 +1,45 @@
+package e2e_test
+
+import (
+	"testing"
+	"time"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	consoleapi "github.com/openshift/console-operator/pkg/api"
+	"github.com/openshift/console-operator/pkg/testframework"
+)
+
+func TestManaged(t *testing.T) {
+	client := testframework.MustNewClientset(t, nil)
```

Comment:
Do we assume each test in isolation, ie should call `err = waitForOperator()` when they start?

Reply:
hmm good question. I would say that initial check should be enough, if we are only going to manipulate with the state of the console-operator. On the other hand it can't  do any harm to check each time?

Reply:
I think we should probably strive to write each e2e test in isolation.

Reply:
sorry for the confusion... I've just realize that the `err = waitForOperator()` that's called in the TestMain function, will be run for each test :)